src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsBaseObject.java

```diff
@@ -789,4 +803,18 @@ public boolean isMiddlePrefooterOnPage() {
   public void verifyMiddlePrefooterAdPresent() {
     verifyAdVisibleInSlot(MIDDLE_PREFOOTER_CSS_SELECTOR, middlePrefooter);
   }
+
+
+  public void switchToFliteSyntheticSuperheroesAd(){
+    driver.switchTo().frame(FLITE_SYNTHETIC_SUPERHEORES_AD_FIRST_IFRAME);
+    driver.switchTo().frame(FLITE_SYNTHETIC_SUPERHEORES_AD_SECOND_IFRAME);
+    driver.switchTo().frame(driver.findElement(By.className(FLITE_SYNTHETIC_SUPERHEORES_AD_THIRD_IFRAME)));
+  }
+
+  public void moveMouseToFliteButton(WebElement button){
+    switchToFliteSyntheticSuperheroesAd();
+    Actions action = new Actions(driver);
+    action.moveToElement(button).perform();
+    driver.switchTo().defaultContent();  }
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Move this closing curly brace to the next line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3ARightCurlyBraceStartLineCheck)